reporter/src/funTest/kotlin/TestData.kt

```diff
@@ -112,6 +112,14 @@ val ORT_RESULT = OrtResult(
                                     id = Identifier("NPM:@ort:declared-license:1.0")
                                 )
                             )
+                        ),
+                        Scope(
+                            name = "devDependencies",
```

Comment:
The `"devDependencies"` is not excluded by default. You must also define a scope exclude then below the paths excludes in line 80.